automation/bootstrap.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -e
+
+PKGS="epel-release git gcc python-devel lvm2 docker"
+yum install -y $PKGS
+rpm -V --nomode $PKGS
+systemctl start docker.service
+yum install -y python2-pip
+rpm -V python2-pip
+pip install --upgrade -r /vagrant/automation/requirements.txt
```

Comment:
So doe this install paclkages into the system or into the user directory?

Also its a good idea to let pip upgrade itself before installing anything else with it

Reply:
pip installs to the system directory of the VM
added pip to the requirements.txt file